CHANGES.rst

```diff
@@ -53,6 +53,10 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Change ``Table.columns.keys()`` and ``Table.columns.values()`` to both return
```

Comment:
Is the duplicate entry intentional?

Reply:
Yes.  It's definitely an API change.  I put it in the new features just to give it more visibility, but I can see the point for taking it out of new features. Either way is fine by me.